src/main/java/com/example/delivery_system/controller/CarriageController.java

```diff
@@ -0,0 +1,43 @@
+package com.example.delivery_system.controller;
+
+import com.example.delivery_system.entity.Carriage;
+import com.example.delivery_system.service.CarriageService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+public class CarriageController {
+    private CarriageService carriageService;
+
+    @Autowired
+    public CarriageController(CarriageService carriageService) {
```

Comment:
The constructor can be generated by Lombok by @RequiredArgsConstructor annotation